src/components/PostsContainer/Post.js

```diff
@@ -25,7 +24,7 @@ const Post = props => {
           src={props.post.imageUrl}
         />
       </div>
-      <LikeSection />
+      <LikeSection likes={props.likes} setLikes={props.likes + 1} />
```

Comment:
Remember that your `setLikes` should be a function like `setLikes={() => setLikes(likes +1)}`